### PR TITLE
Property support

### PR DESCRIPTION
**Pull request by piclane**
- Add property getter support
- Change to get hostname variable from property

### CODE REVIEW

File: src/main/java/jp/co/dwango/logback/SlackWebhookAppender.java [hunk 1]
```diff
@@ -190,9 +180,22 @@ private void initScriptEngine() throws ScriptException {
...
+        function.append(      "var lcvo = event.getLoggerContextVO(),");
+        function.append(          "properties = lcvo.getPropertyMap();");
+        function.append(      "return function(name) {");
+        function.append(        "if(!name && name !== 0) {");
+        function.append(          "return 'Property_HAS_NO_KEY';");
+        function.append(        "}");
+        function.append(        "name = '' + name;");
+        function.append(        "var value = properties.get(name);");
+        function.append(        "return value != null ? value : java.lang.System.getProperty(name);");
+        function.append(      "};");
```
Comment (piclane) [inline src/main/java/jp/co/dwango/logback/SlackWebhookAppender.java:193]: The property function implementation is based on [PropertyConverter](https://github.com/qos-ch/logback/blob/master/logback-classic/src/main/java/ch/qos/logback/classic/pattern/PropertyConverter.java) of logback. This is equivalent to [property conversion](https://logback.qos.ch/manual/layouts.html#property) of PatternLayout.

File: src/main/java/jp/co/dwango/logback/SlackWebhookAppender.java [hunk 2]
```diff
@@ -190,9 +180,22 @@ private void initScriptEngine() throws ScriptException {
...
+        function.append(    "})(), color, emoji,");
+        function.append(    "hostname = property('HOSTNAME'),");
```
Comment (piclane) [inline src/main/java/jp/co/dwango/logback/SlackWebhookAppender.java:195]: changed the hostname to use the [HOSTNAME property](https://logback.qos.ch/manual/configuration.html#hostname).

**Review by xuwei-k — APPROVED** [review]

**Comment by piclane** [comment]
もし異論ないようなら3/30(金)にマージしちゃおうと思います！

**Comment by xuwei-k** [comment]
これも入れてリリースしてしまおう

**Pull request merged**